packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -40,13 +44,49 @@ const Duration kShortRequestTimeout = const Duration(seconds: 5);
 
 /// A connection to the Dart VM Service.
 class VMService {
-  VMService._(this._peer, this.httpAddress, this.wsAddress, this._requestTimeout) {
+  VMService._(this._peer, this.httpAddress, this.wsAddress, this._requestTimeout,
+      ReloadSources reloadSources) {
     _vm = new VM._empty(this);
     _peer.listen().catchError(_connectionError.completeError);
 
     _peer.registerMethod('streamNotify', (rpc.Parameters event) {
       _handleStreamNotify(event.asMap);
     });
+
+    if (reloadSources != null) {
+      _peer.registerMethod('reloadSources', (rpc.Parameters params) async {
+        const int kInvalidParams = -32602;
+        const int kServerError = -32000;
```

Comment:
Just use `rpc_error_code.SERVER_ERROR` -- we're already importing `'package:json_rpc_2/error_code.dart' as rpc_error_code`